.travis.yml

```diff
@@ -1,29 +1,31 @@
 language: dart
 
-dart:
-  - stable
-  - dev
-  - 2.7.2
-
-before_script:
-  - pub run build_runner build --delete-conflicting-outputs
-  - git diff --exit-code
-  - pub run dart_dev analyze
-  - pub run dart_dev format --check
-  - pub run dependency_validator --no-fatal-pins -i analyzer,build_runner,build_web_compilers,built_value_generator
-  # Run Analyzer Plugin checks
-  - cd tools/analyzer_plugin && dart tool/add_over_react_override.dart && pub get
```

Comment:
Actually this OR is missing the `dart tool/add_over_react_override.dart` call, @aaronlademann-wf